app/views/feeds/_form_distribution.html.erb

```diff
@@ -13,7 +13,7 @@
           <% data = {confirm_field_target: "field", confirm_with: with, confirm_delete: delete} %>
           <%= form.text_field :url, data: feed.url_was.present? ? data : {} %>
           <%= form.label :url %>
-          <%= field_link podcast.url if podcast.url.present? %>
+          <%= field_link feed.url if feed.url.present? %>
```

Comment:
it does make sense for this to be the feed url, but was there a reason this used to be the podcast url?

Reply:
This was the bug #883 - the `podcast.url` is one of those fields that's proxied to `podcast.default_feed.url`.  So this link would be correct _only_ if you're looking at the default feed.

(I probably originally copied/pasted this code from somewhere else, and forgot to change this line?)